client/app/lib/accessibility.ts

```diff
@@ -0,0 +1,21 @@
+export function srNotify(text: string, expiry = 1000, container = document.body): void {
```

Comment:
Some documentation on what this does would be nice (I can see this being reused!).

Reply:
Can we use object-style args instead? With two consecutive optionals, you can get calls like `srNotify('test', null, container)`.

Reply:
Could be nice to return a `remove` callback (so if the user blurs really fast, the message is immediately removed).